tests/unit_tests/migrations/viz/dual_line_to_mixed_chart_test.py

```diff
@@ -68,30 +69,4 @@ def test_migration() -> None:
 
 
 def upgrade_downgrade(source, target) -> None:
```

Comment:
This is legacy, but this method name is somewhat confusing, i.e., I'm not _entirely_ sure what it does.

Reply:
Removed 👍🏼